From c9d239490d47d5dd3d7d7b8b7d9007171c5f60ce Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sun, 11 Oct 2015 15:27:09 +0200
Subject: [PATCH] configure.ac: use AC_CONFIG_MACRO_DIR

Instead of having to explicitly pass -I./scripts when running aclocal,
use the AC_CONFIG_MACRO_DIR() macro in configure.ac. This allows to
use "autoreconf" normally, without any hacks.

Submitted upstream at https://github.com/jgaeddert/liquid-dsp/pull/15.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 bootstrap.sh | 2 +-
 configure.ac | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/bootstrap.sh b/bootstrap.sh
index 640e01d..eb4894e 100755
--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -27,7 +27,7 @@
 #
 
 rm -f config.cache aclocal.m4
-aclocal -I./scripts
+aclocal
 autoconf
 autoheader
 #automake --foreign --add-missing
diff --git a/configure.ac b/configure.ac
index a9ad1d7..3b1ba68 100644
--- a/configure.ac
+++ b/configure.ac
@@ -25,6 +25,7 @@
 
 AC_INIT([liquid-dsp],[1.2.0],[support@liquidsdr.org])
 AC_CONFIG_SRCDIR([src/libliquid.c])
+AC_CONFIG_MACRO_DIR([scripts])
 
 # permit auxiliary scripts directory (e.g. config.sub, config.guess, install-sh)
 AC_CONFIG_AUX_DIR(scripts/)
-- 
2.6.1

